pkg/services/pluginsintegration/pluginstore/store.go

```diff
@@ -104,7 +105,7 @@ func (s *Service) SecretsManager(ctx context.Context) *plugins.Plugin {
 
 // plugin finds a plugin with `pluginID` from the registry that is not decommissioned
 func (s *Service) plugin(ctx context.Context, pluginID string) (*plugins.Plugin, bool) {
-	p, exists := s.pluginRegistry.Plugin(ctx, pluginID)
+	p, exists := s.pluginRegistry.Plugin(ctx, pluginID, "") // version is not required since Grafana only supports single versions of a plugin
```

Comment:
I am not sure why `pluginstore` is the only one that _doesn't_ set the version. Shouldn't the rest of `pluginsintegration` follow the same approach?

Reply:
This is the only place now in the codebase where we don't have the version information available. All other places depend on the store itself which is to be seen as the "public" registry, where the registry is more of an internal thing to plugin management. Not sure that explains well enough - but does that make sense?

Reply:
okay, makes sense